include/CL/opencl.hpp

```diff
@@ -7762,6 +7771,35 @@ class CommandQueue : public detail::Wrapper<cl_command_queue>
         return err;
     }
 
+    inline cl_int enqueueReadBufferRect(
```

Comment:
Minor: an explicit `inline` is not required for any member functions defined in the class body.  For consistency and clarity I think we should remove it, both here and for all of the other added member functions in the class body.  (Note, the non-member functions added outside of the class body should still explicitly be marked `inline`!)

```suggestion
    cl_int enqueueReadBufferRect(
```

Reply:
That's absolutely true. I've removed unnecessary inlines and resolved conflicts. Thanks!